packages/swapper/src/index.ts

```diff
@@ -6,3 +6,5 @@ export * from './thorchain-utils'
 export * from './cowswap-utils'
 export * from './safe-utils'
 export * from './swappers/CowSwapper'
+export * from './swappers/JupiterSwapper'
```

Comment:
Unused

```suggestion
```

Reply:
It's used again in [2de18a2](https://github.com/shapeshift/web/pull/8433/commits/2de18a2c00a8ee7874b60ba277000ad720443b90)